leaf-ui/rappid-extensions/LinkInspector.js

```diff
@@ -201,10 +201,19 @@ var LinkInspector = Backbone.View.extend({
         $('option').show(); // Clear the previous selection
         if (begin == 'and' || begin == 'or'){
             $('option.B').hide(); // Hide options incompatible with and/or selection such as +,-,++S, etc
+            if($('#link-type-end :selected').attr('class') == 'B'){ //if user later chose an option in the beginning but the end is incompatible with and/or (ex. +,-,++S)
```

Comment:
For all these comments can you try to make what they are doing a bit clearer, (aka - if end selection is incompatible with new begin selection, clear end). And use the format // Space and capital letter to start the comment